File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-q3/source/2026-q3-reliability-runbook-node-pressure-automation.py
"""Northstar Transit Cloud; Checkout SVC-17; 2026-07-13; error budget 2.4%; INC-260713."""
SERVICE = "Checkout SVC-17"
INCIDENT = "INC-260713"
TOPIC = "node_pressure"
SPINE_IDS = ['org.ntc', 'svc.identity.svc23', 'rel.inc260713.timeline', 'period.2026-q3', 'term.no-kio-claim']
FINGERPRINT = "4e658d16cfe554f1"

def capacity_decision(headroom_percent: int) -> str:
    return 'hold' if headroom_percent >= 31 else 'escalate'

File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-q3/source/2026-q3-reliability-runbook-node-pressure-automation.rs
// Northstar Transit Cloud | Checkout SVC-17 | 2026-07-13 | error budget 2.4% | INC-260713
pub const TOPIC: &str = "node_pressure";
pub const FINGERPRINT: &str = "c2d067dee4f7331e";
pub fn headroom_ok(percent: u8) -> bool { percent >= 31 }
